com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -74,6 +74,13 @@ float4 _ZBufferParams;
 // w = 1.0 if camera is ortho, 0.0 if perspective
 float4 unity_OrthoParams;
 
+// scaleBias.x = flipSign
+// scaleBias.y = scale
+// scaleBias.z = bias
+// scaleBias.w = unused
+uniform float4 _ScaleBias;
```

Comment:
This will cause a conflict for shaders that use `Shaders/Utils/Fullscreen.hlsl`. I think XR team added that as utility/helper to handle fullscreen passes.

Reply:
I removed that parameter from Fullscreen.hlsl. It includes core so and therefore Input as well.